xmodule/js/src/video/037_video_transcript_feedback.js

```diff
@@ -1,31 +1,38 @@
 (function(define) {
-    // eslint-disable-next-line lines-around-directive
-    'use strict';
-
     // VideoTranscriptFeedbackHandler module.
-    define(
-        'video/037_video_transcript_feedback.js', ['underscore'],
-        function(_) {
-            var VideoTranscriptFeedbackHandler;
 
+        'use strict';
+
+        define('video/037_video_caption.js', ['underscore'],
+        function(_) {
             /**
-             * Video Transcript Feedback control module.
-             * @exports video/037_video_transcript_feedback.js
-             * @constructor
-             * @param {jquery Element} element
-             * @param {Object} options
+             * @desc VideoTranscriptFeedbackHandler module exports a function.
+             *
+             * @type {function}
+             * @access public
+             *
+             * @param {object} state - The object containing the state of the video
+             *     player. All other modules, their parameters, public variables, etc.
+             *     are available via this object.
+             *
+             * @this {object} The global window object.
+             *
              */
-            VideoTranscriptFeedbackHandler = function(element, options) {
+
+            var VideoTranscriptFeedbackHandler = function(state) {
                 if (!(this instanceof VideoTranscriptFeedbackHandler)) {
-                    return new VideoTranscriptFeedbackHandler(element, options);
+                    return new VideoTranscriptFeedbackHandler(state);
                 }
 
-                _.bindAll(this, 'sendPositiveFeedback');
-                _.bindAll(this, 'sendNegativeFeedback');
+                _.bindAll(this, 'sendPositiveFeedback', 'sendNegativeFeedback', 'onHideLanguageMenu',
+                    'destroy'
+                );
 
-                this.container = element;
+                this.state = state;
+                this.currentTranscriptLanguage = this.state.lang;
```

Comment:
what value will this have if the user hasn't selected a transcript yet?

Reply:
If the video has at least 1 transcript, that's going to be the selected transcript to provide feedback if we send a request. If it has more than 1 transcript, the first transcript in the dropdown is going to be the selected one.

Reply:
and if there is no transcript selected?

Reply:
When the video is loaded, it automatically selects the first transcript language in the dropdown despite the captions not being enabled. We can update this later if it's not the expected behavior.